Modules/Loadable/Segmentations/Logic/vtkImageGrowCutSegment.h

```diff
@@ -19,8 +19,8 @@ class VTK_SLICER_SEGMENTATIONS_LOGIC_EXPORT vtkImageGrowCutSegment : public vtkI
   /// Set input seed label volume (input 1)
   void SetSeedLabelVolume(vtkImageData* labelImage) { this->SetInputData(1, labelImage); }
 
-  /// Set mask volume (input 2). Optional.
-  /// If this volume is specified then only those regions outside the mask (where mask has zero value)
+  /// Set mask volume (input 2).
```

Comment:
Have you intentionally removed "optional" word? It is important that this mask is not required.

Reply:
That is the point of this PR. That input is not optional. If it is not set, the output is an all zeros image.

Reply:
That's a bug. Can you check if changing this line:

https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Segmentations/Logic/vtkImageGrowCutSegment.cxx#L144

to 

    if (maskLabelVolumePtr != nullptr)

fixes the issue?

Reply:
No, it doesn't.

Possible correction: the output might be just a default-created `vtkImageData`, not an all-zeros image when mask is not set.

Reply:
Anyway, it's just a small bug that I would rather see fixed then having it as a documented limitation. If you use this class in some project anyway then it may worth spending 5-10 minutes with debugging why the mask is not completely ignored when it is not set.

Reply:
Now that I know it is supposed to work, I might do that.